pkg/csi/controller_server.go

```diff
@@ -1464,3 +1470,16 @@ func (cs *ControllerServer) validateHostSnapLabels(hostSnap *snapshotv1.VolumeSn
 
 	return nil
 }
+
+func patchHostClusterName(pvc *corev1.PersistentVolumeClaim, hostClusterName string) {
+	// label the pvc with guest cluster name if we discovered the host cluster name
+	if hostClusterName == "" {
+		logrus.Warnf("hostClusterName is empty, cannot patch PVC %s with guest cluster name", pvc.Name)
```

Comment:
could we move this warning to the controller register? it warns once and could be enough for debugging